app/views/evaluator_submission_assignments/_submission_summary.html.erb

```diff
@@ -0,0 +1,42 @@
+<div
+  class="usa-summary-box bg-base-lightest border-1px border-base-dark radius-0"
+  role="region"
+  aria-labelledby="summary-box-key-information"
+>
+  <div class="usa-summary-box__body">
+    <div class="display-flex flex-column tablet:flex-row tablet:flex-align-center">
+      <div class="tablet:flex-fill">
+        <div class="display-flex flex-align-center margin-bottom-2 tablet:margin-bottom-0">
+          <div class="margin-right-2">
+            <span class="font-sans-3xl text-primary text-bold"><%= @assigned_submissions.count %></span>
+          </div>
+          <div>
+            <h3 class="usa-summary-box__heading text-primary margin-bottom-05" id="summary-box-key-information">
+              Assigned Submissions
+            </h3>
+            <p class="usa-summary-box__text text-primary-darker text-bold">
+              Evaluations due by <%= @phase.end_date.strftime('%m/%d/%Y') %>
```

Comment:
evaluations are due by the form's `closing_date`. there are a few other stats collection examples on #280, although there are subtle differences in the two components. 
```suggestion
              Evaluations due by <%= @phase.evaluation_form.closing_date.strftime('%m/%d/%Y') %>
```